lms/djangoapps/instructor_task/tests/test_integration.py

```diff
@@ -19,7 +19,7 @@
 from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
 from django.test.utils import override_settings
 from django.urls import reverse
-from mock import patch
+from mock import patch  # lint-amnesty, pylint: disable=reimported
```

Comment:
This import can be removed as `patch` is already being imported by `unittest.mock` on line # 14